app/assets/v2/js/grants/new.js

```diff
@@ -149,21 +149,6 @@ const init = () => {
           // trusted relayer
           web3.utils.toChecksumAddress(data.trusted_relayer)
         ];
-      } else if ($('#contract_version').val() == 0) {
```

Comment:
Ready to retire the v0 deploy? I think I'm cool with it...

Reply:
YASS